ote_sdk/ote_sdk/entities/dataset_item.py

```diff
@@ -446,6 +446,8 @@ def __deepcopy__(self, memo):
         for name, value in vars(self).items():
             if "__roi_lock" in name:
                 setattr(clone, name, Lock())
+            elif "__annotation_scene" in name:
```

Comment:
Please update docstring accordingly, specifying annotation scene property is not deep-copied.

Reply:
Added description.